worksheets/Accessibility Manual Testing Criteria.md

```diff
@@ -0,0 +1,55 @@
+# Manual a11y Testing Pass/Fail Criteria
+
+Manual accessibility testing often comes down to subjective reasoning. If something doesn’t feel right, it probably isn’t, and should be logged as an error. Your answers will help determine the accessibility heat index score for the application or site being tested.
+
+## Screen readers - 5 or more errors
```

Comment:
consider writing out as five

Reply:
I realize there is pass/fail criteria, but setting something some baseline ideas as to what a typical starting score might be would be helpful.